services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -94,6 +99,7 @@ export const DocumentTable = ({
     });
   };
   const documents = parseDocuments(props.documents ?? []);
+  const projectGuid = props.projectGuid;
```

Comment:
Document table is used for more than project documents. Cannot rely on project guid being defined. Once Alex's changes with entity_title and so on are merged in, would try following a similar pattern. No references to project should be in this file.

Reply:
removed.